code/modules/mob/living/carbon/human/ai/brain/ai_brain.dm

```diff
@@ -92,9 +92,11 @@ GLOBAL_LIST_EMPTY(human_ai_brains)
 
 	if(!currently_busy && primary_weapon && current_target && !currently_firing && COOLDOWN_FINISHED(src, fire_overload_cooldown) && primary_weapon.has_ammunition())
 		currently_busy = TRUE
-		if(get_dist(tied_human, current_target) > gun_data.maximum_range)
+		var/target_futile = HAS_TRAIT(current_target, TRAIT_INCAPACITATED)
+		if(get_dist(tied_human, current_target) > gun_data.optimal_range || target_futile)
 			if(!has_ongoing_action(/datum/ongoing_action/approach_target) && !in_cover)
-				ADD_ONGOING_ACTION(src, /datum/ongoing_action/approach_target, current_target, gun_data.maximum_range)
+				var/walk_distance = target_futile ? gun_data.minimum_range : gun_data.optimal_range
```

Comment:
Will this cause them to walk into (possible) enemy fire to try and kill a downed member?

Reply:
Nah, when AI is being shot it resets it's current_target to the firer ref, so to try and stay away from new threat when we'll implement retreating on min_range